x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/quick_create_job.ts

```diff
@@ -198,6 +198,10 @@ export class QuickLensJobCreator extends QuickJobCreatorBase {
     bucketSpan: string,
     layerIndex?: number
   ) {
+    // @TODO: ask ML team to check if ES|QL query here is ok
+    if (isOfAggregateQueryType(chartInfo.query)) {
```

Comment:
This check is probably unnecessary as `esql` queries are already blocked in our `isCompatible` function when registering the action.

https://github.com/elastic/kibana/blob/7a98aa176d6dcb3b850b5b9ae2dcd48e7c7ec0cb/x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/utils.ts#L189

It looks like our exiting check could do with being changed to use `isOfAggregateQueryType`

Reply:
Would it be ok to address it as a follow up by your team?

Reply:
sure yeah

Reply:
Created a follow up issue for this: https://github.com/elastic/kibana/issues/199605